### PR TITLE
Allow receiving old messages after rejoining a pubchat

### PR DESCRIPTION
**Pull request by rasom**
needed for https://github.com/status-im/status-react/issues/10362 

status: ready

### CODE REVIEW

File: protocol/message_handler.go [hunk 1]
```diff
@@ -355,7 +355,7 @@ func (m *MessageHandler) HandleChatMessage(state *ReceivedMessageState) error {
 	}
 
 	// If deleted-at is greater, ignore message
-	if chat.DeletedAtClockValue >= receivedMessage.Clock {
+	if chat.DeletedAtClockValue >= receivedMessage.Clock && chat.ChatType != ChatTypePublic {
```
Comment (cammellos) [inline protocol/message_handler.go]: There's already a method I believe:
`chat.Public()`

**Review by cammellos — APPROVED** [review]
Minor style change, we could also have a test to make sure that even if deletedAt is set you receive messages in a public chat, not that critical but would be nice

File: protocol/message_handler.go (hunk 1 above)
Comment (rasom) [reply protocol/message_handler.go]: fixed, thanks

**Comment by rasom** [comment]
@cammellos added test. I thought about this but was too lazy.

**Pull request merged**